packages/_private/angular-demo/src/app.component.html

```diff
@@ -1,4 +1,13 @@
-<syn-header label="synergy-design-system/angular Form Demo">
+<syn-header label="synergy-design-system/angular Components Demo">
+  <a
+    class="custom-logo"
+    href="/"
+    routerLink="/"
+    slot="logo"
+  >
+    <syn-icon name="logo-color" library="system" />
+  </a>
```

Comment:
This (using an `<a/>` tag around a syn-icon in the `logo` slot) makes the header look wrong with safari. There is too much space at the left side of the logo: 
![image](https://github.com/user-attachments/assets/5386241b-5c7c-49e4-81de-fe9ad7e294d4)

Reply:
Fixed. We did not get this initially as Storybook was not tested with Safari when we created the header.